device/tencent/x86_64-tencent_tcs8400-r0/TCS8400-24CC8CD/port_config.ini

```diff
@@ -0,0 +1,33 @@
+# name          lanes                                alias               index      speed
+Ethernet1       25,26,27,28                          Eth200GE0/1         0          200000
```

Comment:
Are we going with 0 based indexing? I think in SONiC we moved to 1-based indexing of the ports.

Reply:
Our device's front panel port index starts at 1. This configuration file does not conform to the guide, but it is functionally fine, so let's modify it

Reply:
As you reconsidering the numbering, please check the logical interfaces numbers too.
In case the port breakout is supported, there are no numbers left for the broken out ports. It might be good to number the logical ports as Ethernet0, Ethernet4, Ethernet8 etc rather than Ethernet1, Ethernet2, Ethernet3 etc. 
Pls check other platforms logical interfaces numbering pattern.

Reply:
This platform mainly completes the adaptation of s3ip framework, At present, we do not support port breakout, so we plan not to reserve ports for broken out ports at first.
We plan to support port breakout in future upgrades, then re-index the port.

Reply:
The code has been updated, please help to continue the review